deforestation_api/routers/healthcheck.py

```diff
@@ -30,12 +30,22 @@ def lossyear_data_available():
 health.add_check(lossyear_data_available)
 
 
-@router.get("/ready", tags=["health"])
+@router.get(
+    "/ready",
+    summary="Check if this service is ready to receive requests",
+    description="Returns a message describing the status of this service",
+    tags=["health"],
```

Comment:
I don't think it's necessary to include these tags here since you already added them in the `router`. Same with the liveness-endpoint